i18n/fr/building.md

```diff
@@ -185,24 +185,24 @@ pacman -Su
 $ git clone --recursive https://github.com/monero-project/kovri
 ```
 
-### 2. Compilez kovri et les dépendances *submodule* en une commande
+### 2. Compilez kovri et les dépendances des sous-modules en une commande
 ```bash
 $ make  # pour réduire le temps de compilation : `make -j <available CPU cores>`. Par exemple : `make -j4`
 ```
 
-### 3. Installation
+### 3. Installez
 ```bash
 $ make install
 ```
 
 - Les utilisateurs DOIVENT lancer `make install` pour les nouvelles installations
 - Les développeurs DEVRAIENT lancer `make install` après compilation
 
-### Les autres options que vous pouvez utiliser à la place de l'étape 2 :
+### Autres options que vous pouvez utiliser à la place de l'étape 2 :
 
 Pour une liste complète des options, voir les *targets* dans le fichier [Makefile](https://github.com/monero-project/kovri/blob/master/Makefile).
 
-#### Notes
+#### Remarque
```

Comment:
Remarques?

Reply:
Not sure, there's only one Note after all...

Reply:
Indeed - let's leave it as is then!